strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/storage/repository/aws/AwsObjectFactory.java

```diff
@@ -0,0 +1,39 @@
+package org.carlspring.strongbox.storage.repository.aws;
+
+import org.carlspring.strongbox.storage.repository.CustomConfiguration;
+
+import javax.xml.bind.JAXBElement;
+import javax.xml.bind.annotation.XmlElementDecl;
+import javax.xml.bind.annotation.XmlRegistry;
+import javax.xml.namespace.QName;
+
+/**
+ * @author carlspring
+ */
+@XmlRegistry
+public class AwsObjectFactory
+{
+
+
+    @XmlElementDecl(name = "aws-configuration", scope = AwsConfiguration.class)
+    JAXBElement<AwsConfiguration> createAwsConfiguration(AwsConfiguration configuration)
+    {
+        return new JAXBElement<AwsConfiguration>(new QName("", "aws-configuration"),
+                                                 AwsConfiguration.class,
+                                                 configuration);
+    }
+
+    public static AwsConfiguration createAwsConfiguration()
```

Comment:
I made it static, because it was complaining there was no static factory method for this.